gateway/src/main/java/com/mx/path/gateway/configuration/AccessorStackConfigurator.java

```diff
@@ -148,7 +151,10 @@ public final Accessor buildFromNode(String clientId, ObjectMap node) {
     AccessorScope accessorScope = determineAccessorScope(node, accessorType);
     Class<?> proxyType = AccessorProxyMap.get(accessorScope.getName(), Accessor.getAccessorBase(accessorType));
 
-    return new ClassHelper().buildInstance(Accessor.class, proxyType, configurationBuilder.build(), accessorType);
+    Accessor accessor = new ClassHelper().buildInstance(Accessor.class, proxyType, configurationBuilder.build(), accessorType);
+    invokeAfterInitializeMethods(accessor.getClass());
+
+    return accessor;
```

Comment:
I thought we were going to make the afterInitialize method static, so we don't need to create an instance. I think that will be safer. I'm not sure what the impact would be of generating an instance of every accessor outside of the context of a prepared gateway.

Reply:
Fixed.